backend/src/main/java/com/empire/Noble.java

```diff
@@ -12,6 +12,14 @@
 	double unrest;
 	private double experience;
 
+	enum Action {
+		SOOTHE,
+		LEVY,
+		CONSCRIPT,
+		OTHER;
+	}
+	transient Action action = Action.OTHER;
```

Comment:
Curious about a few things here.  Firstly, why make this transient?  Secondly, does gson know not to include this member based on it being transient?  Thirdly, is there a gson annotation that could more explicitly signal to exclude action during ser-des?

Reply:
"transient" indicates that a field should not be serialized/deserialized, e.g. by Gson (or any other serializer/deserializer) - we make it transient specifically so that it is not serialized (Gson will respect this.) There may be a specific Gson annotation (or a policy or type adapter or something we can register), but those are all more verbose.